lib/infra_conversion_throttler.rb

```diff
@@ -1,15 +1,24 @@
 class InfraConversionThrottler
   def self.start_conversions
     pending_conversion_jobs.each do |ems, jobs|
-      running = ems.conversion_hosts.inject(0) { |sum, ch| sum + ch.active_tasks.size }
+      running = ems.conversion_hosts.inject(0) { |sum, ch| sum + ch.active_tasks.count }
```

Comment:
Looks like it may be tricky treating all conversion_hosts the same.

I'll try and think of a way, but adding together `vm_conversion_hosts` values and `host_conversion_hosts` values may be the next best thing.

encapsulating this and putting it into ems may at least make this method look good.